src/common/config.h

```diff
@@ -25,6 +25,7 @@
 #define OSD_REP_SPLAY   1
 #define OSD_REP_CHAIN   2
 
+#define VAL1 1
```

Comment:
Variable name can be changed!!! your suggestions would be good..
But using constant in code ... I believe is no Good..